src/compass/utils/geo_metadata.py

```diff
@@ -265,3 +265,181 @@ def to_file(self, dst, fmt:str):
+                              self.range_pixel_spacing,
+                              {'description':'range pixel spacing',
+                               'units':'meters'})
+
+        def poly1d_to_h5(group, poly1d_name, poly1d):
+            '''Write isce3.core.Poly1d properties to hdf5
+
+            Parameters
+            ----------
+            group: h5py.Group
+                h5py Group to store poly1d parameters in
+            poly1d_name: str
+                Name of Poly1d whose parameters are to be stored
+            poly1d: isce3.core.Poly1d
+                Poly1d ojbect whose parameters are to be stored
+            '''
+            poly1d_group = group.require_group(poly1d_name)
+            add_dataset_and_attrs(poly1d_group, 'order', poly1d.order,
+                                  {'description': 'order of the polynomial'})
+            add_dataset_and_attrs(poly1d_group, 'mean', poly1d.mean,
+                                  {'description': 'mean of the polynomial'})
+            add_dataset_and_attrs(poly1d_group, 'std', poly1d.std,
+                                  {'description': 'standard deviation of the polynomial'})
+            add_dataset_and_attrs(poly1d_group, 'coeffs', poly1d.coeffs,
+                                  {'description': 'coefficients of the polynomial'})
+        poly1d_to_h5(metadata_group, 'azimuth_fm_rate', self.azimuth_fm_rate)
+        poly1d_to_h5(metadata_group, 'doppler', self.doppler)
+
+        add_dataset_and_attrs(metadata_group, 'range_bandwidth',
+                              self.range_bandwidth,
+                              {'description':'bandwidth used during processing',
+                               'units':'Hz'})
+        add_dataset_and_attrs(metadata_group, 'polarization',
+                              np.string_(self.polarization),
+                              {'description': 'burst polarization'})
+        add_dataset_and_attrs(metadata_group, 'burst_id',
+                              np.string_(self.burst_id),
+                              {'description': 'unique ID for burst'})
+        add_dataset_and_attrs(metadata_group, 'platform_id',
+                              np.string_(self.platform_id),
+                              {'description': 'Sentinel platform ID (S1, S2)'})
+
+        center_lon_lat = np.array([val[0] for val in self.center.coords.xy])
+        add_dataset_and_attrs(metadata_group, 'center',
+                              center_lon_lat,
+                              {'description': 'coordinates of center of burst',
+                               'units': 'degrees'})
+
+        # list of lon, lat coordinate tuples (in degrees) representing burst border
+        border_group = metadata_group.require_group('border')
+        border_x, border_y = self.border.exterior.coords.xy
+        add_dataset_and_attrs(border_group, 'x', border_x,
+                              {'description': 'list boundary x-coordinates',
```

Comment:
Can we be more descriptive? Something like "List of coordinates in the X-direction describing the burst perimeter on the ground". Same applies for the Y-direction